src/ethers/saga.ts

```diff
@@ -0,0 +1,42 @@
+import BigNumber from 'bignumber.js'
+import { FeeInfo } from 'src/fees/saga'
+import { buildSendTx } from 'src/send/saga'
+import { chooseTxFeeDetails } from 'src/transactions/send'
+import { call } from 'typed-redux-saga'
+
+// Helper function to call chooseTxFeeDetails for send transactions (aka transfer contract calls)
+// using parameters that are not specific to contractkit
+export function* getSendTxFeeDetails({
+  recipientAddress,
+  amount,
+  tokenAddress,
+  feeInfo,
+  encryptedComment,
+}: {
+  recipientAddress: string
+  amount: BigNumber
+  tokenAddress: string
+  feeInfo: FeeInfo
+  encryptedComment?: string
+}) {
+  const celoTx = yield* call(
+    buildSendTx,
+    tokenAddress,
+    amount,
+    recipientAddress,
+    encryptedComment || ''
+  )
+  // It may never be worth it for us to port this fee calculation logic to ethers since it is so celo-specific
```

Comment:
does this comment assume we stay on contractkit forever for celo? contractkit is not going to be supported anymore soon so idk if that makes sense to assume

Reply:
I'll rephrase, probably a good idea to never say 'never' anyways in these kinds of comments